<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialDebugAssistantClass</class>
 <widget class="QWidget" name="SerialDebugAssistantClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>509</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>787</width>
    <height>509</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>787</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="3,1,0,0">
   <item row="10" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>接收区配置</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>61</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>接收模式</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>文本编码</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QComboBox" name="comboRxMode">
         <item>
          <property name="text">
           <string>HEX模式</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>文本模式</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboRxEncode">
         <item>
          <property name="text">
           <string>GBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>发送区配置</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>19</y>
        <width>61</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>发送模式</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>文本编码</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>19</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QComboBox" name="comboTxMode">
         <item>
          <property name="text">
           <string>HEX模式</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>文本模式</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboTxEncode">
         <item>
          <property name="text">
           <string>GBK</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="7" column="0" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>接收区</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>9</y>
        <width>563</width>
        <height>287</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>563</width>
        <height>241</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="0,0">
       <item row="1" column="1">
        <widget class="QPushButton" name="key_COM_Tx2">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>清空接收区</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>545</width>
           <height>222</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="7" column="1" rowspan="3" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>串口配置</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>61</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>串口号</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>校验位</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>操作</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>131</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_available_COM"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_baudRate">
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_dataBits">
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_stopBits">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_checkBits">
         <item>
          <property name="text">
           <string>无</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>奇校验</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>偶校验</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="key_open_COM">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="10" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>发送区</string>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>563</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="6" alignment="Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignBottom">
        <widget class="QPushButton" name="key_COM_Tx1">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>清空发送区</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="6">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="key_COM_Tx">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>41</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
